docs/insiders/getting-started.md

```diff
@@ -185,4 +185,17 @@ pipeline), you can build your documentation project with the following lines:
 mkdocs build --config-file mkdocs.insiders.yml
 ```
 
+!!! warning "Alternative syntax"
+    
+    If you plan to split `plugins` or `markdown_extensions` between the two files, you must use
+    alternative key / value syntax for these sections.  The above insiders file would then look like:
+    ``` yaml
+    INHERIT: mkdocs.yml
```

Comment:
Could we maybe provide two code blocks each for each `mkdocs.yml` file (base + Insiders overrides) which just adds the `social` plugin? The other plugins are non-Insiders anyway. This would be a more self-contained example for the user to understand how to do overrides while keeping the plugin configuration of the base configuration. I say we could just add the `social` plugin as an example. Otherwise, users might start to mix syntaxes, or is that possible?

Reply:
So essentially something like:

`mkdocs.yml`

``` yaml
plugins:
  search: {}
  tags: {}
```

`mkdocs.insiders.yml`

``` yaml
INHERIT: mkdocs.yml
plugins:
  social {}
```

Please use the content tabs pattern for this, as in [other parts of the documentation](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#custom-admonitions).

Reply:
Thanks for your patience with my slow updates - I just pushed the requested changes!